libs/designer-ui/src/lib/tokenpicker/tokenpickerfooter.tsx

```diff
@@ -96,7 +100,12 @@ export function TokenPickerFooter({
     try {
       currExpression = ExpressionParser.parseExpression(expression.value);
     } catch (ex) {
-      setExpressionEditorError(invalidExpression);
+      if (expression.value.includes('"')) {
```

Comment:
I wonder if there's a way we can generically add something to expression parser instead. this doesn't seem very scalable to other error types.

Reply:
hello, I have reworked it so that is throws a different error in the parser if the problem is due to misused double quotes, let me know what you think!